jhu/delphi_jhu/run.py

```diff
@@ -4,8 +4,10 @@
 This module should contain a function called `run_module`, that is executed
 when the module is run with `python -m MODULE_NAME`.
 """
-from datetime import datetime
+from datetime import datetime as dt
 from itertools import product
+import datetime
```

Comment:
I don't think this `import datetime` is needed.

Reply:
So you can probably also delete the alias of `datetime.datetime` to `dt` if you would like.

Reply:
I'm still using it on line 156 (datetime.datetime.now())

Reply:
right but you have already imported `datetime.datetime` as `dt`

Reply:
Oh nvm, I see what you mean. Removed.